src/Native/AnyOS/brotli/include/brotli/port.h

```diff
@@ -103,6 +103,10 @@ To apply compiler hint, enclose the branching condition into macros, like this:
 #else
 #define BROTLI_ENC_API __declspec(dllimport)
 #endif  /* BROTLIENC_SHARED_COMPILATION */
+#elif !defined(_WIN32)
+#define BROTLI_COMMON_API __attribute__ ((__visibility__ ("default")))
+#define BROTLI_DEC_API __attribute__ ((__visibility__ ("default")))
+#define BROTLI_ENC_API __attribute__ ((__visibility__ ("default")))
```

Comment:
Are we going to need to redefine these every time we update our brotli source?  Can we do without this change?

Reply:
I opened https://github.com/google/brotli/issues/687 upstream and maintainer has fixed it in master. They have plans to make a new release this week (today or tomorrow), which will land at: https://github.com/google/brotli/releases. At which point we can update brotli to the versioned tag (instead of random commit hash) and also close https://github.com/dotnet/corefx/issues/30249.

Is this a reasonable plan? I can follow up once @eustas and co. release Brotli vNext.

Reply:
Sounds reasonable.  Thanks.